package.json

```diff
@@ -21,53 +21,60 @@
     "test": "tests"
   },
   "scripts": {
-    "build": "ember build",
-    "lint": "eslint .",
+    "build": "ember build --environment=production",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*",
+    "lint:js": "eslint .",
     "start": "ember serve",
-    "test": "tape node-tests/**.js",
-    "test:all": "ember try:each"
+    "test": "npm-run-all lint:* test:*",
+    "test:node": "tape node-tests/**.js",
+    "test:ember": "ember test",
+    "test:ember-compatibility": "ember try:each"
```

Comment:
Running the `test` script will now run both the ember and the node tests

I'll be adding tests when adding the `renderStory()` function